app/models/question.rb

```diff
@@ -3,4 +3,8 @@ class Question < ActiveRecord::Base
   has_and_belongs_to_many :templates
   validates :content,  presence: true, uniqueness: { case_sensitive: false }
   validates :topic,    presence: true
+  validates :level, 	 presence: true
+
+  POSSIBLE_LEVELS = ['beginner', 'good', 'strong']
```

Comment:
if we nave only 3 possible options - then we should validate inclusion
http://guides.rubyonrails.org/active_record_validations.html
2.6